### PR TITLE
[Policy] Needs-Manual-Merge

### PR DESCRIPTION
**Pull request by mdanish-kh**
cc @stephengillie / @Trenly

This is for https://github.com/microsoft/winget-pkgs/pull/110985#issuecomment-1618869308

@stephengillie for creating a new label `Needs-Manual-Merge` with a color of your choice 😄 
Description: The pull request requires a manual merge from a repository maintainer.

-----

### CODE REVIEW

**Review by Trenly — APPROVED** [review]
Approving pending label creation

**Comment by stephengillie** [comment]
We have `Merge-Ready`, would that be suitable here?

**Comment by mdanish-kh** [comment]
> We have `Merge-Ready`, would that be suitable here?

The description of Merge-Ready `The pull request can be merged now` doesn't really fit with a PR that is blocked and requires a maintainer to manually merge it. 

I see you added `Needs-Manual-Merge`, that looks more accurate

**Review by stephengillie — APPROVED** [review]

**Pull request merged**
